docs/developers/guides/run-a-node/use-binary.mdx

```diff
@@ -56,6 +56,19 @@ Download the genesis file and Besu configuration file.
 
 </Tabs>
 
+:::warning[Important: enable linea-sequencer]
+
+You must install and enable the `linea-sequencer` Besu plugin if you are offering RPC services or 
+hosting nodes. Enabling the plugin is a requirement for using certain API methods, such as 
+[`linea_estimateGas`](/developers/reference/api/linea-estimategas).
+
+Please see our [instructions for installing the plugin](./linea-sequencer).
+
+If you are running a node to follow the chain activity, rather than submit transactions, you don't 
+need to install `linea-sequencer`.
+
+:::
+
```

Comment:
Remove goerli from https://doc-zk-evm-git-linea-sequencer-infura-web.vercel.app/developers/guides/run-a-node/use-binary

Reply:
Remove full stop at end of Step 4

Reply:
Good spot on both counts, thank you